content/community.html

```diff
@@ -112,22 +112,22 @@ <h2 id="mailing-lists">Mailing Lists</h2>
   <tbody>
     <tr>
       <td>User support and questions mailing list<br /><strong>user</strong>@gearpump.incubator.apache.org</td>
-      <td><a href="mailto:user-subscribe@gearpump.incubator.apache.org?subject=Subscribe">Subscribe</a></td>
-      <td><a href="mailto:user-unsubscribe@gearpump.incubator.apache.org?subject=Unsubscribe">Unsubscribe</a></td>
-      <td><a href="mailto:user@gearpump.incubator.apache.org">Compose</a></td>
+      <td><a href="&#109;&#097;&#105;&#108;&#116;&#111;:&#117;&#115;&#101;&#114;&#045;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#064;&#103;&#101;&#097;&#114;&#112;&#117;&#109;&#112;&#046;&#105;&#110;&#099;&#117;&#098;&#097;&#116;&#111;&#114;&#046;&#097;&#112;&#097;&#099;&#104;&#101;&#046;&#111;&#114;&#103;&#063;&#115;&#117;&#098;&#106;&#101;&#099;&#116;&#061;&#083;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;">Subscribe</a></td>
```

Comment:
These are strange.  THese are ascii codes for href value. 
But this works: the link still produces proper "mailto: xxx" links.
I am not sure what caused the change... And I am not sure how to force the tools to leave it be...

But like I said, it works anyhow.